### DISCUSSION TITLE
关于数字加粗的问题。

### DISCUSSION THREAD

**Discussion by qzxuhui** [tuna/thuthesis#744] [Q&A] [answered]
在表示向量的时候，数字需要加粗处理，例如求解线性方程组
```
\bm{Ax}-\bm{b}=\bm{0}
```
其中`\bm{0}`代表零向量。

在模板编译的时候，好像`\bm{0}`和`0`显示出来的0效果一样，没有加粗效果。
请问应该如何处理，使得数字也有加粗效果呢？谢谢。

在thusetup中选择`math-font  = lm`和`math-font  = xits`都有这个问题。

谢谢。

**Comment by qzxuhui** [comment]
用`\mathbf{0}`可以正常加粗

**Comment by zepinglee** [comment, marked as answer]
模板中使用了 `unicode-math` 宏包，所以应当使用 `\symbf` 表示粗体。

`\bm` 能编译是因为模板做了兼容性处理，将 `\bm` 定义为 `\symbfit`。它作用于字母时总是使用粗斜体，但是对于数字而言就没有加粗的效果了。
